.idea/misc.xml

```diff
@@ -3,9 +3,8 @@
   <component name="DesignSurface">
     <option name="filePathToZoomLevelMap">
       <map>
-        <entry key="app/src/main/res/layout/add_habit_dialog_box.xml" value="0.5" />
+        <entry key="..\:/Users/jas98/AndroidStudioProjects/habitSmasher/app/src/main/res/layout/add_habit_dialog_box.xml" value="0.12760416666666666" />
```

Comment:
This **shouldn't** be here. This path is local to your computer.

- [ ] remove this